testsuite/openshift/client.py

```diff
@@ -32,12 +32,12 @@ class OpenShiftClient:
     def __init__(self, project: str, api_url: str = None, token: str = None, kubeconfig_path: str = None):
         self._project = project
         self._api_url = api_url
-        self.token = token
+        self._token = token
```

Comment:
why?

Reply:
This token is `NoneType` most of the time. An openshift token is needed to request Prometheus metrics. New property was introduced to fetch the real token. https://github.com/Kuadrant/testsuite/blob/c76fcea6d8501c0843e1b23e562dfbca7531b08d/testsuite/openshift/client.py#L60-L64